timeless/views.py

```diff
@@ -27,3 +27,57 @@ def get(self, *args, **kwargs):
             if result is None:
                 abort(404)
         return "", HTTPStatus.OK
+
+
+class GenericView(views.View):
```

Comment:
@vladarefiev let's force guys to provide template name in declarative way, if they want more complex cases, they will override `get_template_name` method
```python
class GenericView(views.View):
    template_name = None

    def get_template_name(self):
        """ Setup of template name """
        return self.template_name
```